addon/components/frost-select.js

```diff
@@ -203,6 +203,20 @@ export default Ember.Component.extend({
     }
   },
 
+  /* Ember.didInsertElement method */
+  didInsertElement () {
```

Comment:
I would highly recommend changing the dropdown container to be a component and adding this logic there.  The handler only needs to be present when the container is showing